supportedBackends/tes/src/main/scala/cromwell/backend/impl/tes/TesAsyncBackendJobExecutionActor.scala

```diff
@@ -123,6 +138,13 @@ class TesAsyncBackendJobExecutionActor(override val standardParams: StandardAsyn
     )
   }
 
+  private def blobPathWithoutPrefix(blobPath: BlobPath, prefix: Path): String = {
```

Comment:
I wonder if it might make sense to have a small test for this?

Reply:
Yes! Added (though... ignored 😞 ).